src/Compilers/Extension/CompilerPackage.cs

```diff
@@ -100,9 +95,14 @@ private void WriteMSBuildFiles(string packagePath, string hiveName)
             }
             catch (Exception e)
             {
+                var msg = 
+$@"{e.Message}
+
+To reload the Roslyn compiler package, close Visual Studio and any MSBuild processes, then restart Visual Studio.";
+
```

Comment:
It looks like this message is shown to a user and, therefore, should be localizable. Perhaps we should open a separate issue for this.